src/ensemble/ensemble_solutions.jl

```diff
@@ -37,10 +37,11 @@ end
 """
 $(TYPEDEF)
 """
-struct EnsembleSummary{T,N,Tt,S,S2,S3,S4} <: AbstractEnsembleSolution{T,N,S}
+struct EnsembleSummary{T,N,Tt,S,S2,S3,S4,S5} <: AbstractEnsembleSolution{T,N,S}
   t::Tt
   u::S
   v::S2
+  med::S5
```

Comment:
This was the minimal change but the result is a bit weird. Maybe some types should be reordered, but I was not 100% sure if anything else might rely on the current type signature of EnsembleSummary.

Reply:
Nothing else should rely on it.